website/src/pages/index.js

```diff
@@ -120,7 +120,7 @@ const Showcase = () => {
   }
   const showcases = siteConfig.customFields.showcases.map((user) => (
     <a href={user.infoLink} key={user.infoLink} target="_blank">
-      <img className="user-logo" src={user.image} alt={user.caption} />
+      <img className="user-logo" src={'https://cdn.jsdelivr.net/gh/apache/apisix-website@master/website/static/' +  user.image} alt={user.caption} />
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/112404902-2d2eba00-8d4c-11eb-8f05-6674f7af9e95.png)

https://deploy-preview-270--apisix.netlify.app/

Broken, right?

Reply:
yup. missing the `/img` path, I will fix it later 😀